src/sentry/workflow_engine/models/workflow.py

```diff
@@ -32,6 +34,8 @@ class Workflow(DefaultFieldsModel, OwnerModel, JSONConfigBase):
 
     created_by_id = HybridCloudForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete="SET_NULL")
 
+    frequency = models.IntegerField(default=DEFAULT_FREQUENCY)
```

Comment:
fyi, we removed this from the other db migration because this only applies to issue alerts, instead we can make a subclass an ad the `frequency` to the `config` property.

Here's a link to that discussion: https://github.com/getsentry/sentry/pull/80710#discussion_r1847052467 / why it wasn't added.

Reply:
oh i missed that, thanks!